docs/languages/swift.md

```diff
@@ -0,0 +1,30 @@
+# Swift
+
+## Install Syntax Highlighting
+```vim
+:TSInstall swift
+```
+
+## Install Language Server
+
+```vim
+:LspInstall sourcekit
+```
+
+
+## Configure Lunarvim
+
+The Lsp, and formatting does not work out of the box. To fix this, do the following:
```

Comment:
```suggestion
The `sourcekit` language server is disabled (overridden) by default because of the conflict with `clangd`
```

I also feel like we shouldn't let `clangd` run at the same time, but I'm not sure.

Reply:
Running `lvim`(with my custom config forked from abz's) in the swift compiler project enables only these things for me: 
![image](https://user-images.githubusercontent.com/48220549/155297604-67f1d444-56cc-4a80-a852-9f810084e10f.png)

Clangd runs normally in other projects.